doc/install/stampede.tex

```diff
@@ -13,32 +13,30 @@
 
 
 
-Setting up ASPECT on TACC Stampede is much easier than on previous systems since
+\noindent Setting up ASPECT on TACC Stampede is much easier than on previous systems since
```

Comment:
Justified indentation is a personal preference. No problem to change use the defaults.